### PR TITLE
Completed T15

### PR DESCRIPTION
**Pull request by Ajay07Metamation**

### CODE REVIEW

File: Training/Program.cs [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿//Program to check Isogram
+using static System.Console;
+bool breakouter = false;
```
Comment (rajeevramanTMM) [inline Training/Program.cs:3]: Rewrite using a function call.

File: Training/Program.cs [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
+Write ("Enter a String : ");
+String letter = ReadLine () ?? "";
```
Comment (rajeevramanTMM) [inline Training/Program.cs:5]: Use string data type not class

File: Training/Program.cs [hunk 3]
```diff
@@ -1,7 +1,20 @@
...
+//Using for loop
+for (int ch = 0; ch < letter.Length - 1; ch++) {
+   for (int ch1 = ch + 1; ch1 < letter.Length; ch1++) {
+      if (letter[ch] == letter[ch1]) {
+         WriteLine ("It is not an Isogram");
+         breakouter = true;
+         break;
+      }
+   }
+   if (breakouter == true) break;
```
Comment (rajeevramanTMM) [inline Training/Program.cs:15]: No need to check true explicitly.

File: Training/Program.cs [hunk 4]
```diff
@@ -1,7 +1,20 @@
...
+}
+if (breakouter == false) WriteLine ("It is an Isogram");
```
Comment (rajeevramanTMM) [inline Training/Program.cs:17]: Handle this situtaion.
image

File: Training/Program.cs [hunk 5]
```diff
@@ -1,7 +1,20 @@
...
+// Using LINQ operator
```
Comment (rajeevramanTMM) [inline Training/Program.cs:18]: Is this a reference code for testing the method if so please provide a naming and print it. This is printing two time has shown in the image above which is confusing.

**Review by rajeevramanTMM — CHANGES_REQUESTED** [review]

File: Training/Program.cs (hunk 5 above)
Comment (Ajay07Metamation) [reply Training/Program.cs:17]: All changes done in latest commit

File: Training/Program.cs [hunk 6]
```diff
@@ -1,7 +1,22 @@
...
-        }
-    }
-}
\ No newline at end of file
+﻿//Program to check Isogram
+using static System.Console;
+Write ("Enter a String : ");
+String letter = ReadLine () ?? "";
+WriteLine(IsIsogram(letter));
+string IsIsogram(string s) {
```
Comment (rajeevramanTMM) [inline Training/Program.cs:6]: The name IsIsogram represents a boolean return type but here you returning a string type. Please change the return type or the method name. Please leave a line in between a  method and a section of code and formatting to be applied.

**Review by rajeevramanTMM — CHANGES_REQUESTED** [review]

File: Training/Program.cs (hunk 6 above)
Comment (Ajay07Metamation) [reply Training/Program.cs]: All changes and formatting done in latest commit

File: Training/Program.cs [hunk 7]
```diff
@@ -1,7 +1,23 @@
...
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿//Program to check Isogram
+using static System.Console;
+Write ("Enter a String : ");
+string letter = ReadLine () ?? "";
+if (letter.All (char.IsLetter) && letter.Length >= 1)
```
Comment (rajeevramanTMM) [inline Training/Program.cs:5]: Can you please check the length condition first and use any operator instead of All. All will be checking the full string.

**Review by rajeevramanTMM — CHANGES_REQUESTED** [review]

**Pull request closed without merging**